styles/mixins/_atoms.scss

```
@@ -0,0 +1,35 @@
+@use 'sass:color';
+@use 'sass:list';
+
+@use '../variables' as var;
+
+$-color-border: var.$color-border;
+$-color-shadow: var.$color-shadow;
+
+/**
+ * Add border styles for all element sides (default) or a specific single side
+ *
+ * @param {'left'|'right'|'bottom'|'top'} [$side]
+ */
+@mixin border($side: null) {
+  $valid-sides: left, right, bottom, top;
+  $border-property: border;
+  @if list.index($valid-sides, $side) {
+    $border-property: border-#{$side};
+  }
+  #{$border-property}: 1px solid $-color-border;
+}
+
+/**
+ * Style a shadow off of the element. `$active` variant is larger, darker,
+ * intended for hovered/active elements.
+ *
+ * @param {boolean} [$active]
+ */
+@mixin shadow($active: false) {
+  @if $active {
+    box-shadow: 0px 2px 3px 0px color.scale($-color-shadow, $alpha: -85%);
+  } @else {
+    box-shadow: 0 1px 1px color.scale($-color-shadow, $alpha: -90%);
```

Added `px` for consistency:

```suggestion
    box-shadow: 0px 1px 1px color.scale($-color-shadow, $alpha: -90%);
```

or maybe adding the `spread-radius` to be more explicit?

```suggestion
    box-shadow: 0px 1px 1px 0px color.scale($-color-shadow, $alpha: -90%);
```